java/test/src/main/java/org/ray/api/test/CrossLanguageInvocationTest.java

```diff
@@ -48,26 +53,32 @@
 
   @Test
   public void testCallingPythonFunction() {
-    RayObject res = Ray.callPy(PYTHON_MODULE, "py_func", "hello".getBytes());
+    RayObject res = Ray.call(
+        new PyRemoteFunction(PYTHON_MODULE, "py_func", Object.class),
+        "hello".getBytes());
     Assert.assertEquals(res.get(), "Response from Python: hello".getBytes());
   }
 
   @Test
   public void testPythonCallJavaFunction() {
-    RayObject res = Ray.callPy(PYTHON_MODULE, "py_func_call_java_function", "hello".getBytes());
+    RayObject res = Ray.call(
```

Comment:
Update all the `RayObject res` to `RayObject<byte[]> res`.

Reply:
Fixed.